.github/ISSUE_TEMPLATE/troubleshooting.md

```diff
@@ -0,0 +1,6 @@
+<!-- ⚠️⚠️STOP⚠️⚠️ -- PLEASE READ! -->
+
+We use GitHub to track feature requests and bug reports.
```

Comment:
@carlosroman Is this boilerplate from the repo template? I think we'd use a more friendly tone here... But it's OK for the time being if we're in a hurry.

Reply:
Yep, from template.